File: backend/Multibanca.Domain.Models/Multibanca/cierre_copias_notaria.cs
using Common.Domain.Models;
using System;

namespace Multibanca.Domain.Models.Multibanca
{
    public class cierre_copias_notaria : base_auditoria
    {
        public int id_cierre_copias_notaria { get; set; }
        public long id_expediente { get; set; }
        public bool? enviar_a_reparo { get; set; }
        public string? observaciones { get; set; }
    }
}

File: frontend/src/features/actividades/cierre_copias_notaria/models/cierre_copias_notaria.ts
export interface CierreCopiasNotaria {
  id_cierre_copias_notaria: number;
  id_expediente: number;
  enviar_a_reparo: boolean;
  observaciones?: string | null;

  is_active: boolean;
  row_status: boolean;
  created_by: number;
  created_date: string;
  modified_by?: number | null;
  modified_date?: string | null;
}
